include/xten/Dialect/XTenNN/IR/XTenNNOps.td

```diff
@@ -101,9 +101,9 @@ def XTenNN_QuantizeOp: XTenNN_Op<"quantize", [
             Elementwise,
             Pure,
             SameOperandsAndResultShape]> {
-  let summary = "Quantizes a float32 tensor to a signed or unsigned integer tensor of given width.";
+  let summary = "Quantizes a float32 tensor to a signless or unsigned integer tensor of given width.";
   let description = [{
-    Quantizes a given float32 tensor into a signed or unsigned integer tensor of given width.
+    Quantizes a given float32 tensor into a signless or unsigned integer tensor of given width.
```

Comment:
Just mention we are using the assumption that signless integers describe signed integers. Meaning if the type is `!unsigned` then its signed.